langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -110,6 +140,11 @@ public isolated function concat((xml|string)... xs) returns xml = @java:Method {
 
 # Returns a string giving the expanded name of an xml element.
 #
+# ```ballerina
+# xml:Element person = xml `<person age="30">John</person>`;
+# string xmlElementName = person.getName();
```

Comment:
We can rename this variable to `name` to maintain the simplicity. wdyt?

Reply:
Shall we use an example in which `name` does not make sense btw. For example, at first glance, this may kinda be misinterpreted to be returning the name of the person.

Reply:
Same for `setName`.

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/240eeaad04c7ba4dc901592cdc6875a656057250